styleshet.css

```diff
@@ -104,5 +108,6 @@ td .content {
     word-break: break-word;
     hyphens: auto;
     padding: 2px;
+	cursor: pointer;
```

Comment:
Not an issue on a touch screen but when you have a cursor you expect that cursor to tell you if you can click what its currently hovering over